### ISSUE TITLE
Median problem has no edge cases

### ISSUE THREAD

**Issue by erp12** [thelmuth/program-synthesis-benchmark-datasets#9]
The `median-edge.*` gzip files appear to be corrupted after #8. I rolled back my local repo to see what was in them before and it was only a header row. The transformation applied in #8 changed the header-only files to empty files.

If the median problem has no edge cases, should we just remove those files from the repo?

EDIT: I am seeing the same thing for number-io

**Comment by thelmuth** [comment]
Yes, deleting those files seems like the reasonable thing to do. I double checked, and those two problems should be the only ones without edge cases.

Want to put in a pull request? If not, I can take care of it sometime.
